test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4243,6 +4243,7 @@ def _load_configuration(self, infile, objtype=None):
                     return False
             conf = sconf[str(objtype)]
             if objtype == MGR_OBJ_SERVER:
+                conf = sconf[str(objtype)]
```

Comment:
you are setting `conf` just above one line, I did not understand why are you doing it here?

Reply:
It was residual part of code added while debugging.